x-pack/tsconfig.json

```diff
@@ -1,31 +1,14 @@
 {
   "extends": "../tsconfig.json",
-  "include": [
-    "common/**/*",
-    "server/**/*",
-    "plugins/**/*",
-  ],
-  "exclude": [
-    "test/**/*"
-  ],
+  "include": ["common/**/*", "server/**/*", "plugins/**/*"],
+  "exclude": ["test/**/*", "plugins/secops/node_modules/react-split-pane"],
```

Comment:
I take it this is the only line that was really added. Everything else an IDE prettier change (or the layout was just annoying you :-P)

Reply:
Yes, that's `prettier` re-formatting the file